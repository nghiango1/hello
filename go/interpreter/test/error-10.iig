// "identifier not found: foobar"
foobar
			
