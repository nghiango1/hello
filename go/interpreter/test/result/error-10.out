ERROR: identifier not found: foobar
